docs/source/blogs/tech_blog/blog10_ADP_Balance_Strategy.md

```
@@ -0,0 +1,368 @@
+
+### Scheduling Strategies for Load Balancing
+
+The fundamental challenge in Attention DP is that ranks can process vastly different token loads within the same iteration, causing the overall execution time to be bottlenecked by the most heavily loaded rank.
+
+#### Baseline: Round-Robin Token Distribution
+
+The conventional approach employs a global load balancing strategy that sorts incoming requests by `num_tokens` and distributes them across ranks using round-robin scheduling, as illustrated in Figure 1. This method achieves reasonable token distribution from a cumulative perspective and effectively reduces token count disparities when all ranks are simultaneously processing context requests.
+
+<div align="center">
+<figure>
+  <img src="./../media/tech_blog10_baseline_round_robin_strategy.png">
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)